src/versionUtils.ts

```diff
@@ -195,21 +195,32 @@ export async function getDownloadURLForCamelKTag(tag : string, platformStr : str
 	if (githubToken) {
 		headers.push(['Authorization', `token ${githubToken}`]);
 	}
-	console.log(tagURL);
-	const res: Response = await fetch(tagURL, { headers: headers });
+	const res: Response = await timeout(9000, fetch(tagURL, { headers: headers }));
```

Comment:
I added a timeout of 9 secs here to make it not wait too long. Maybe we could even reduce more like to 4-5 secs instead until we expect a response to the request.